README.rst

```diff
@@ -109,7 +119,7 @@ https://git-scm.com/docs/git-submodule
 Installing the Images
 =====================
 
-See detailed instructions in doc/howtos/image-install.rst.
+See detailed instructions in `how to install images <doc/howtos/image-install.rst>`__.
```

Comment:
Thanks for the explanation @dbkinder . I OK with two trailing underscores, I just did not know what it meant and I now got educated, thanks :-)